src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ClientRepository.cs

```diff
@@ -18,6 +18,22 @@ public ClientRepository(ISqlConnectionFactory sqlConnectionFactory) : base(sqlCo
         {
         }
 
+        public async Task<bool> MemberHasClientsAsync(Guid memberId)
```

Comment:
Please write test for this repository method.

Reply:
done